package.json

```diff
@@ -25,19 +27,29 @@
     "csv-parser": "^2.3.2",
     "csvtojson": "^2.0.10",
     "eslint": "^6.7.2",
+    "eslint-config-prettier": "^6.10.0",
+    "eslint-plugin-prettier": "^3.1.2",
     "event-stream": "^4.0.1",
     "express": "^4.17.1",
     "express-joi-validation": "^4.0.3",
     "getstream-node": "^1.5.0",
     "http-status-codes": "^1.4.0",
+    "joi": "^14.3.1",
+    "jsonwebtoken": "^8.5.1",
     "mongoose": "^5.7.13",
+    "path": "^0.12.7",
```

Comment:
Why do you need "path" package installed if it's already a part of Node.js as built-in package?

Reply:
resolvde